music/demos/transcription.ts

```diff
@@ -31,6 +31,33 @@ const EXPECTED_NS_URL = `${
     TRANS_CKPT_DIR}/onsets_frames_htk0/MAPS_MUS-mz_331_3_ENSTDkCl.melhtk0-250frames.ns.json`;
 // tslint:enable:max-line-length
 
+function sortNotes(notes: mm.NoteSequence.INote[]) {
+  notes.sort((a, b) => {
+    if (a.startTime < b.startTime) {
+      return -1;
+    }
+    if (a.endTime < b.endTime) {
+      return -1
+    }
+    return a.pitch - b.pitch;
+  });
+}
+
+function notesMatch(a: mm.NoteSequence.INote[], b: mm.NoteSequence.INote[]) {
```

Comment:
Alternatively, this could also be a method on `NoteSequence` itself. Something like `matches`?

Reply:
(you don't have to do this, it's just a suggestion)